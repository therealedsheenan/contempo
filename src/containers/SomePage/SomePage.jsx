// @flow

import React from 'react';

export default () => (
  <div
    className="Styleguide"
    style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', width: '100%', height: '100vh' }}>
    <p style={{ textAlign: 'center', margin: '0' }}>This is another page.</p>
  </div>
);
