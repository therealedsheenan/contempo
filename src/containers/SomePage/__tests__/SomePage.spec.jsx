// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import SomePage from '../SomePage';

test('Home existence', () => {
  const component = renderer.create(<SomePage />).toJSON();

  // correct home component
  expect(component).toMatchSnapshot();
});
