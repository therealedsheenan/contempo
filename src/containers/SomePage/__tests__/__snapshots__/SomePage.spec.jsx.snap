// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home existence 1`] = `
<div
  className="Styleguide"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <p
    style={
      Object {
        "margin": "0",
        "textAlign": "center",
      }
    }
  >
    This is another page.
  </p>
</div>
`;
